src/Components/Admin/UserRoleSelect.js

```diff
@@ -16,6 +18,7 @@ const StyledSelect = styled(Select)`
 function UserRoleSelect({ id, value, onChange }) {
   return (
     <StyledSelect
+      native
```

Comment:
I was having trouble getting the select to show on top of the dialog, unless this was set and `MenuItem`s were `option`s. Let me know if you know of a fix?